tests/kernels/test_sphere_heat_kernel.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+import torch
+import lab as B
+
+import geometric_kernels.torch
+from geometric_kernels.spaces.hypersphere import Hypersphere
+from geometric_kernels.kernels.geometric_kernels import MaternKarhunenLoeveKernel
+
+
+_TRUNCATION_LEVEL = 10
+_NU = 2.5
+
+
+def manifold_laplacian(manifold, x, egrad, ehess):
+    r"""
+    Computes the manifold Laplacian of a given function at a given point x.
```

Comment:
This looks like a useful utility, maybe promote it to the core codebase as opposed to the test?

Reply:
Sure, that may be better! Where would you see it in the codebase?